CONTRIBUTING.md

```diff
@@ -533,9 +533,7 @@ Scala has proven the most viable way to do it, as long as you keep the following
 1. If the underlying Scala code requires an `ExecutionContext`, make the Java API take an `Executor` and use
    `ExecutionContext.fromExecutor(executor)` for conversion.
 
-1. Make use of `scala-java8-compat` conversions, see [GitHub](https://github.com/scala/scala-java8-compat)
-   (eg. `scala.compat.java8.FutureConverters` to translate Futures to `CompletionStage`s).
-   Note that we cannot upgrade to a newer version scala-java8-compat because of binary compatibility issues.
+1. Use `org.apache.pekko.util.FutureConverters` to translate Futures to `CompletionStage`s.
```

Comment:
![image](https://github.com/apache/incubator-pekko/assets/501740/0313e88f-045d-41ed-a762-4ccdb19a545a)

It's private.

Reply:
@mdedetrich Should we make it public?

Reply:
Specifically about the documentation, I think this is fine because its in `CONTRIBUTING.md`, i.e. its intended for people working on Pekko codebase not Pekko users.

Reply:
This particular notes are for people contributing code to Pekko project. They are not about public APIs.

Users writing code that uses Pekko: if they are using Java APIs of Pekko, they should not be getting Scala Futures as return types - so they won't need to convert them.

Reply:
Ha, I forgot that, I was thinking a user guide:)